server-spi/src/main/java/org/keycloak/models/RealmModel.java

```diff
@@ -29,11 +29,9 @@
 import org.keycloak.storage.client.ClientStorageProviderModel;
 import org.keycloak.storage.role.RoleStorageProvider;
 import org.keycloak.storage.role.RoleStorageProviderModel;
```

Comment:
By keycloak's convention, it might be better not to modify codes that is not related to the PR itself.

Reply:
Yes, I will change it.